docs/source/conf.py

```diff
@@ -113,45 +113,21 @@
     "smartquotes",
 ]
 
+myst_heading_anchors = 3
+
 # Tells MyST to treat URIs beginning with these prefixes as external links.
 # Links that don't begin with these will be treated as internal cross-links.
 myst_url_schemes = ("http", "https", "mailto")
 
 # -- Options for myst_nb -----------------------------------------------------
 
-# Needed to explicitly set render priority for the `doctest` mode.
-# https://myst-nb.readthedocs.io/en/latest/use/formatting_outputs.html#render-priority
-nb_render_priority = {
-    "html": (
-        "application/vnd.jupyter.widget-view+json",
-        "application/javascript",
-        "text/html",
-        "image/svg+xml",
-        "image/png",
-        "image/jpeg",
-        "text/markdown",
-        "text/latex",
-        "text/plain",
-    ),
-    "doctest": (
-        "application/vnd.jupyter.widget-view+json",
-        "application/javascript",
-        "text/html",
-        "image/svg+xml",
-        "image/png",
-        "image/jpeg",
-        "text/markdown",
-        "text/latex",
-        "text/plain",
-    ),
-}
```

Comment:
Nice cleaning

Reply:
Again, this seems to be a case of adding default parameters to our config. If it's default, lets get rid of it! Less code means less maintenance burden 😄